developer-docs-site/docs/concepts/coin-and-token/aptos-token-overview.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Aptos Token Standard Overview"
+id: "aptos-token-overview"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Aptos Token Standard Overview
+
+:::tip Aptos token standard
+Also see the official token standard [Aptos Token](/concepts/coin-and-token/aptos-token.md).
+:::
+
+## Introduction
+One of the major application of blockchain is token, which requires a standard defining
+the pattern a token must comply with. The Aptos team and communities also have been moving
+fast and steadily to support various projects and creators to launch their tokens on aptos
+blockchain. As a new L1 blockchain equipped with Move, a relatively new language exclusively
+designed for the safety of smart contract, our token standard inevitably differs from other
+blockchains more or less. In this doc, we will briefly cover the comparison between aptos
+ethereum and solana token standard.
+
+### Account Model
+In order to understand tokens, we have to compare the account model across different
+blockchains. Here is a brief summary.
+
```

Comment:
I'm not sure this adds much, but :shrug:

Reply:
The reason is when I created the doc, this is the prerequisite from my own experience.